### PR TITLE
awesome manifesto

### PR DESCRIPTION
**Pull request by n1trux**
as stated in #207, we should discuss the guidelines about what is awesome here.

### CODE REVIEW

File: awesome.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# The awesome manifesto
+
+If you want your list to be included on `awesome`, you should really
```
Comment (arthurvr) [inline awesome.md]: > If you want your list to be included on `awesome`, you should really  just include awesome stuff.

This makes me think this is a list of rules to be included in the list. It should just be a list of tips/guidelines about how to make your lists more awesome.

File: awesome.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Please be sure that you do your best to keep the list grammatically correct. Pay attention to Markdown formatting errors as well, only accept pull requests with proper formatting.
+
+## Choose an appropriate license
+
+See [Choose a License](http://choosealicense.com/no-license/) for more details. Keep in mind that if you haven't selected a license, it basically means the people are not allowed to reproduce, distribute or create derivative works.
+
+## Include contribution guidelines
+
+People who are contributing to your list should have a clear understanding how they should do so. You could copy a contributing.md from another awesome repository you like.
```
Comment (arthurvr) [inline awesome.md]: This also needs to explain what the contributing guidelines should guide about: formatting of the list, order, ...

Comment (n1trux) [inline awesome.md]: what else?

Comment (arthurvr) [inline awesome.md]: These => https://github.com/sindresorhus/awesome/blob/master/contributing.md

**Comment by sindresorhus** [comment]
// @aleksandar-todorovic @h4cc

File: awesome.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Make sure your list covers a certain scope and nothing else.
+Link to other awesome lists if you think they already cover a certain subject good enough.
+
+## Pay attention to grammar
+
+Please be sure that you do your best to keep the list grammatically correct. Pay attention to Markdown formatting errors as well, only accept pull requests with proper formatting.
+
+## Choose an appropriate license
+
+See [Choose a License](http://choosealicense.com/no-license/) for more details. Keep in mind that if you haven't selected a license, it basically means the people are not allowed to reproduce, distribute or create derivative works.
```
Comment (h4cc) [inline awesome.md]: Is the link target correct? "no-license" looks like the wrong target.

Comment (aleksandar-todorovic) [inline awesome.md]: @h4cc Yup, you're right, it should link to the homepage. I originally wanted to explain what happens when you don't select a license and I forgot to change it once I paraphrased the paragraph.

File: awesome.md (hunk 2 above)
Comment (n1trux) [inline awesome.md]: >You could copy a contributing.md from another awesome repository you like.

Isn't this enough?

Comment (arthurvr) [inline awesome.md]: Okay but then link up a good one (as ours, for example)

Comment (n1trux) [inline awesome.md]: did it.

File: awesome.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# The awesome manifesto
+
+If you want your list to be included on `awesome`, you should really just include awesome stuff.
+
+But **what is awesome?**
+
+## Only awesome is awesome.
```
Comment (arthurvr) [inline awesome.md]: The period at the end of this title is inconsistent.

**Comment by n1trux** [comment]
so, does it look finished for the moment? Should we merge it?

@h4cc @aleksandar-todorovic @sindresorhus @arthurvr

**Comment by aleksandar-todorovic** [comment]
One small suggestion:

> You could **just** copy a [contributing.md](contributing.md) from another awesome repository you like **if you don't feel like writing a new one from scratch**.

I bolded the words I think should be added. In my opinion, by leaving it like this, we're kind of encouraging people to just copy that file. In my opinion, that file should be a bit more descriptive and kind of depends on the topic of the list.

**Comment by h4cc** [comment]
I am kind of confused about the explanation "what is awesome". Should the list itself be awesome, or the linked content?
In my case the awesome-elixir list is the current only resource for easy finding any elixir/erlang library out there - thats what makes itself awesome, not every linked content is awesome too. So i have even duplicate entries in there.

A focus on the awesomeness of the list itself would be a better fit, i think.

**Comment by aleksandar-todorovic** [comment]
> Should the list itself be awesome, or the linked content?

Both. It should link only to awesome resources and the list itself needs to be awesome (basically to follow everything from this pull request).

File: awesome.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# The awesome manifesto
+
+If you want your list to be included on `awesome`, you should really just include awesome stuff.
```
Comment (sindresorhus) [inline awesome.md]: > If you want your list to be included on `awesome`, try to only include awesome stuff in your list. After all, it's a curation, not a collection.

File: awesome.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Make sure your list covers a certain scope and nothing else.
+Link to other awesome lists if you think they already cover a certain subject good enough.
+
+## Pay attention to grammar
+
+Please be sure that you do your best to keep the list grammatically correct. Pay attention to Markdown formatting errors as well, only accept pull requests with proper formatting.
+
+## Choose an appropriate license
+
+See [Choose a License](http://choosealicense.com/) for more details. Keep in mind that if you [haven't](http://choosealicense.com/no-license/) selected a license, it basically means the people are *not* allowed to reproduce, distribute or create derivative works.
```
Comment (sindresorhus) [inline awesome.md]: We should recommend a license. I would say [CC0](https://creativecommons.org/publicdomain/zero/1.0/) is a good one.

File: awesome.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+## Include contribution guidelines
+
+People who are contributing to your list should have a clear understanding how they should do so.
+
+If you don't feel like writing one from scratch, you could copy a [contributing.md](contributing.md) from another awesome repository you like.
```
Comment (sindresorhus) [inline awesome.md]: I don't think this is a good idea. Instead we should just recommend reading the one in this repo and make their own from that.

File: readme.md [hunk 8]
```diff
@@ -2,6 +2,9 @@
 
 > A curated list of awesome lists
 
+We did a small document about [what awesome is](awesome.md).
```
Comment (sindresorhus) [inline readme.md]: `[What makes for an awesome list?](awesome.md)`

**Comment by n1trux** [comment]
@strugee because it highlights the dangers of choosing no license.

@sindresorhus incorporated your suggestions.

**Comment by sindresorhus** [comment]
Thanks @n1trux :)

**Pull request closed without merging**